bayes_opt/acquisition.py

```diff
@@ -198,15 +217,28 @@ def _acq_min(self, acq: Callable, bounds: np.ndarray, n_random=10_000, n_l_bfgs_
         if n_random == 0 and n_l_bfgs_b == 0:
             error_msg = "Either n_random or n_l_bfgs_b needs to be greater than 0."
             raise ValueError(error_msg)
+        x_min_r: NDArray[Float] | None
+        x_min_l: NDArray[Float] | None
+        min_acq_r: float
+        min_acq_l: float
```

Comment:
Shouldn't the value of these be clear simply because the functions that assign them are correctly annotated with type hints?

Reply:
As you said, it was unnecessary after adding type hints to `_random_sample_minimize` and `_l_bfgs_b_minimize`.

I'll fix it soon.